examples/java/src/main/java/com/pelias/importer/BostonSchools.java

```diff
@@ -64,7 +64,7 @@ public void process() throws ClientProtocolException, IOException {
 
         // setup connection to Elasticsearch
         TransportClient elasticSearchClient = TransportClient.builder().build().addTransportAddress(
-                new InetSocketTransportAddress(InetAddress.getByName("localhost"), 9300));
+                new InetSocketTransportAddress(InetAddress.getByName("elasticsearch"), 9300));
```

Comment:
I'm pretty sure you want to use port 9200, not 9300. 9300 is generally the port used for internal communication between Elasticsearch nodes for the purposes of managing replication and whatnot. 9200 is where queries usually go. But if it works...